packages/core/src/app/payment/PaymentSubmitButton.tsx

```diff
@@ -14,6 +14,7 @@ interface PaymentSubmitButtonTextProps {
     methodName?: string;
     initialisationStrategyType?: string;
     brandName?: string;
+    isMethodCheckoutStarted?: boolean;
```

Comment:
I understand that this flag indicates whether checkout will be continued or completed.
Can we give it a better name?

Reply:
> I understand that this flag indicates whether checkout will be continued or completed. Can we give it a better name?

What name do you suggest?

Reply:
We can start to place order on checkout page or via SPB. This flag show we start place order on checkout page after click on payment method

Reply:
If we started place order on the checkout page then the button name should be `Continue with PayPal` otherwise `Complete order`

Reply:
Perhaps `isComplete`? We used it a lot in `checkout-js`.

Reply:
done